File: fizzbuzz.c
// FizzBuzz written in C
//
#include <stdio.h>
#include <math.h>

int  main()
{
  int count;
  for (count = 1; count < 101; count++)
  {
    if ((count % 3) == 0)
      printf ("Fizz");

    if ((count % 5) == 0)
      printf ("Buzz");

    if (((count % 5) != 0) && ((count % 3) != 0))
      printf ("%d", count);

    printf ("\n");
  } 
  return 0;
}

File: fizzbuzz.rb
(1..100).each do |number|

  if (number % 3 == 0)
    print "Fizz"
    skipnum = true
  end

  if (number % 5 == 0)
    print "Buzz"
    skipnum = true
  end

  if (!skipnum)
    print number
  end

  print "\n"

end

